debian/config

```diff
@@ -2,6 +2,15 @@
 
 set -e
 
+# There are two questions open:
+
+# wartool -m  is inconsistently documented. It can be easily added here, but is it still part of wartool?
+
+# how should the Expansion Pack (Beyond the Dark Portal) be handled? I've
```

Comment:
So we have 3 scenarios:
1. ToD alone
2. BtDP alone
3. ToD + BtDP

I propose to reduce the installer to:
1. Provide one dataset (ToD alone or BtDP alone)
2. Provide two datasets (ToD + BtDP)

Is that ok?

-- 
Karol Kreński

> Anyway, the expansion CD can be extracted on its own and the game will
> work, but to get both the ToD and BtDP campaigns you need to extract
> first from the base game and then from the expansion CD.
> Alternatively, I'm now building the wargus debian packages with
> StormLib, so the Battle.net Edition should also work and then you get
> both campaigns from one CD.